js/exporter/image_creator.js

```diff
@@ -644,33 +660,50 @@ function strokeElement(context, options, isText) {
     }
 }
 
-function getPattern(context, pattern, shared) {
+function getPattern(context, pattern, shared, parentOptions) {
     const options = getElementOptions(pattern, shared.rootAppended);
     const patternCanvas = imageCreator._createCanvas(options.width, options.height, 0);
     const patternContext = patternCanvas.getContext('2d');
 
-    drawCanvasElements(pattern.childNodes, patternContext, options, shared);
+    const d = drawCanvasElements(pattern.childNodes, patternContext, options, shared);
```

Comment:
Please can you change the name of the variable "d", When I look at line 677, I have to keep an understanding of what "d"